idaes/ui/tests/serialized_boiler_flowsheet.json

```diff
@@ -1,2606 +1 @@
ay": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_outlet_1"}}}], "z": 2}, {"type": "standard.Link", "source": {"id": "side_1_inlet_1", "port": "out"}, "target": {"id": "ECON", "port": "in"}, "router": {"name": "orthogonal", "padding": 10}, "connector": {"name": "normal", "attrs": {"line": {"stroke": "#5c9adb"}}}, "id": "s_side_1_inlet_1", "labels": [{"attrs": {"rect": {"fill": "#d7dce0", "stroke": "white", "stroke-width": 0, "fill-opacity": 0}, "text": {"text": "feed info", "fill": "black", "text-anchor": "left", "display": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_side_1_inlet_1"}}}], "z": 2}, {"type": "standard.Link", "source": {"id": "ECON", "port": "out"}, "target": {"id": "side_2_outlet_1", "port": "in"}, "router": {"name": "orthogonal", "padding": 10}, "connector": {"name": "normal", "attrs": {"line": {"stroke": "#5c9adb"}}}, "id": "s_side_2_outlet_1", "labels": [{"attrs": {"rect": {"fill": "#d7dce0", "stroke": "white", "stroke-width": 0, "fill-opacity": 0}, "text": {"text": "product info", "fill": "black", "text-anchor": "left", "display": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_side_2_outlet_1"}}}], "z": 2}, {"type": "standard.Link", "source": {"id": "side_2_inlet_1", "port": "out"}, "target": {"id": "FSH", "port": "in"}, "router": {"name": "orthogonal", "padding": 10}, "connector": {"name": "normal", "attrs": {"line": {"stroke": "#5c9adb"}}}, "id": "s_side_2_inlet_1", "labels": [{"attrs": {"rect": {"fill": "#d7dce0", "stroke": "white", "stroke-width": 0, "fill-opacity": 0}, "text": {"text": "feed info", "fill": "black", "text-anchor": "left", "display": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_side_2_inlet_1"}}}], "z": 2}, {"type": "standard.Link", "source": {"id": "side_1_inlet_2", "port": "out"}, "target": {"id": "RH", "port": "in"}, "router": {"name": "orthogonal", "padding": 10}, "connector": {"name": "normal", "attrs": {"line": {"stroke": "#5c9adb"}}}, "id": "s_side_1_inlet_2", "labels": [{"attrs": {"rect": {"fill": "#d7dce0", "stroke": "white", "stroke-width": 0, "fill-opacity": 0}, "text": {"text": "feed info", "fill": "black", "text-anchor": "left", "display": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_side_1_inlet_2"}}}], "z": 2}, {"type": "standard.Link", "source": {"id": "RH", "port": "out"}, "target": {"id": "side_1_outlet_1", "port": "in"}, "router": {"name": "orthogonal", "padding": 10}, "connector": {"name": "normal", "attrs": {"line": {"stroke": "#5c9adb"}}}, "id": "s_side_1_outlet_1", "labels": [{"attrs": {"rect": {"fill": "#d7dce0", "stroke": "white", "stroke-width": 0, "fill-opacity": 0}, "text": {"text": "product info", "fill": "black", "text-anchor": "left", "display": "none"}}, "position": {"distance": 0.66, "offset": -40}}, {"attrs": {"text": {"text": "s_side_1_outlet_1"}}}], "z": 2}]}
```

Comment:
Could you format this like the old json was formatted? Either copy and paste this into a json formatter on the web or do `json.dumps(parsed, indent=4, sort_keys=True)`

Reply:
I was wondering how to get the formatting done.

I also need some help with sorting out the issue with Windows tolerance. I have not managed to limit the signficiant figures in the dataframe/json dump yet.